packages/babel-parser/test/fixtures/es2020/import-meta/without-dynamic-import/options.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Since we dynamic import is always enabled, this test can be removed (it was used to test `import.meta` without `import()` support)